### DISCUSSION TITLE
Quarto website for NHS-R Community

### DISCUSSION THREAD

**Discussion by Lextuga007** [quarto-dev/quarto-cli#10958] [Show and Tell]
### Description

This week we moved from the WordPress site for NHS-R Community to Quarto https://nhsrcommunity.com/ which is fully open and published on GitHub https://github.com/nhs-r-community/nhs-r-community. 

I was inspired by the [RWDS](https://realworlddatascience.net/) and [Silvia Canelón's personal blog](https://silviacanelon.com/) to get this up and running.

Also a shout out to dukestatsci for their [GitHub docker image](https://github.com/nhs-r-community/nhs-r-community/blob/main/.github/workflows/publish.yaml) which the site uses. I tried to create a similar docker image for NHS-R Community but it didn't quite work for the geospatial blogs and had to revert back. That's an ongoing project but sums up how the NHS-R Community GitHub is as it's full of ongoing projects 😄

**Comment by cderv** [comment]
Awesome ! Thanks for sharing ! 

Are you ok for us to add your website in Quarto's gallery ? https://quarto.org/docs/gallery/#websites

**Comment by Lextuga007** [comment]
Absolutely would love to see it included! Thanks 😊

**Reply by cderv** [reply]
Great ! It will be live on website soon. Thanks !

**Reply by Lextuga007** [reply]
Oops, I think the subtext was copied from the link just before: `Website for ecological dataviz, analysis & research` it should say `Promoting the use of R and open analytics in health and care`. Thanks so much for adding it though and so quickly - I really appreciate it!

**Reply by cderv** [reply]
I went too fast 🤦 Thank you !

**Reply by Lextuga007** [reply]
Not a problem at all, it's exciting it's been added so quickly!
